### ISSUE TITLE
[new feature] Disable plugin when you open a non Laravel project

### ISSUE THREAD

**Issue by pniaps** [NiclasvanEyk/jetbrains-laravel-make-integration#42]
Thanks for this great plugin.

It would be a good idea if the plugin is disabled when you open a non Laravel project.

Now it shows a ,notification like this ....
```
Could not use artisan binary
Could not execute `php artisan version`!
D:\..........\php.exe D:\.........\artisan "" --version
Could not open input file: D:\.......\artisan
Done!
Usually this means that you need to fix or specify your PHP project interpreter through the settings menu.
```

It's not a Laravel project, so the notification is annoying. On the other hand, this is not a bug, the user should not be bothered.

A possible solution would be that if the `artisan` file does not exist, consider that the project is not Laravel and deactivate the plugin so that the menu entries "File > New", Laravel tool window and other things do not appear.

**Comment by NiclasvanEyk** [comment]
Thanks for the kind words and reporting this issue. This behavior is definitely not intended, as you said, none of the plugin features should be active for non-Laravel projects. 

I'll investigate this and try to push a fix sfter I've found the cause.

**Comment by NiclasvanEyk** [comment]
Should be fixed in the latest 3.1 release. 

You can test it by manually installing the zip file from the release page on GitHub, or wait a few days until Jetbrains approves the update.

**Comment by pniaps** [comment]
Works great, thanks for the quick response.
